Softeq.XToolkit.WhiteLabel/Navigation/IPageNavigationService.cs

```diff
@@ -21,8 +20,8 @@ public interface IPageNavigationService
 
         void GoBack();
 
-        void RestoreState();
-
         NavigateHelper<T> For<T>() where T : IViewModelBase;
+
+        IViewModelBase GetExistingOrCreateViewModel(Type type);
```

Comment:
A good candidate for the move to viewlocator, but we need BackStack,
need discuss